protocol/src/main/java/org/jboss/as/protocol/ProtocolConnectionUtils.java

```diff
@@ -192,7 +192,7 @@ public static Connection connectSync(final ProtocolConnectionConfiguration confi
             saslOptions = new HashMap<>(saslOptions);
             // Drop SASL_DISALLOWED_MECHANISMS which we already handled
             saslOptions.remove(Options.SASL_DISALLOWED_MECHANISMS.getName());
-            mergedConfiguration = mergedConfiguration.useMechanismProperties(saslOptions);
+            mergedConfiguration = mergedConfiguration.useSaslMechanismProperties(saslOptions);
```

Comment:
what's the relation with the YAML support?

Reply:
I guess this is a rebase artefact